e2e-tests/functions

```diff
@@ -4,14 +4,14 @@ GIT_COMMIT=$(git rev-parse HEAD)
 GIT_BRANCH=${VERSION:-$(git rev-parse --abbrev-ref HEAD | sed -e 's^/^-^g; s^[.]^-^g;' | sed -e 's/_/-/g' | tr '[:upper:]' '[:lower:]')}
 API="psmdb.percona.com/v1-12-0"
 IMAGE=${IMAGE:-"perconalab/percona-server-mongodb-operator:${GIT_BRANCH}"}
-IMAGE_PMM=${IMAGE_PMM:-"perconalab/pmm-client:dev-latest"}
+IMAGE_PMM=${IMAGE_PMM:-"percona/pmm-client:2.27.0"}
```

Comment:
I think we shouldn't change this line. It is for tests only and can be redefined via env variable.

Reply:
I will fix it) Ok!